locales/en-US/notes.properties

```diff
@@ -0,0 +1,13 @@
+welcomeTitle=Welcome!
+welcomeText=This is a simple one-page notepad built in to Firefox that helps you get the most out of the web.
+
+emptyPlaceHolder=Take a note…
+
+openingLogin=Opening Login…
+forgetEmail=Forget this Email
+
+syncNotReady=Sorry, Sync is not ready yet, but we will count your click as a vote to hurry it up!
```

Comment:
Side question when trying to localize these strings: is Sync intended as generic synchronization, or Firefox Sync? Same for disableSync later in the file